bot/ordersActions.js

```diff
@@ -53,8 +53,8 @@ const createOrder = async (i18n, bot, user, {
       description: buildDescription(i18n, {
         user,
         type,
-        amount,
-        fiatAmount,
+        amount: numberFormat(fiatCode, amount),
+        fiatAmount: isNaN(fiatAmount) ? fiatAmount : numberFormat(fiatCode, fiatAmount),
```

Comment:
Why if isNaN(fiatAmount) then fiatAmount ?

Reply:
If the user sends a flatAmount, such as an interval (Eg. 1000-10000), the format function can´t apply to the included numbers. The interval fiatAmount equals a string (Not a Number).

Reply:
in that case you need to apply the format function both values of fiatAmount